types/docker-modem/tslint.json

```
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "await-promise": false
+    }
```

```suggestion
    "extends": "dtslint/dt.json"
```
why do you need this exclusion from DT rules?